### ISSUE TITLE
Uncaught SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by Patresko** [jonataswalker/ol-contextmenu#251]
I get this error when html page is loaded. 

```
<script type=""text/javascript"" src=""/wcsfiles/js/openlayer/ol.js></script>
<script type=""text/javascript"" src=""/wcsfiles/js/openlayer/contextmenu/ol-contextmenu.js></script>
```

If I tried open example from source, there is an error as well

`Uncaught ReferenceError: ContextMenu is not defined`

**Comment by jonataswalker** [comment]
@Patresko Are you using any kind of bundler (Vite, Webpack)?

If not, use `dist/ol-contextmenu.iife.js` or CDN:

```
<script src=" https://cdn.jsdelivr.net/npm/ol-contextmenu@5.1.4/dist/ol-contextmenu.iife.min.js "></script>
<link href=" https://cdn.jsdelivr.net/npm/ol-contextmenu@5.1.4/dist/ol-contextmenu.min.css " rel="stylesheet">
```

**Comment by jonataswalker** [comment]
Hopefully solved by #260
